node/src/components/gossiper/tests.rs

```diff
@@ -384,9 +328,76 @@ impl reactor::Reactor for Reactor {
                 self.contract_runtime
                     .handle_event(effect_builder, rng, *event),
             ),
+            Event::Chainspec(_request) => todo!(),
         }
     }
 
+    fn new(
```

Comment:
It would be a simpler diff if `new()` wasn't moved.

Reply:
Un-moved